sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcQuerySuite.scala

```diff
@@ -164,6 +165,14 @@ class OrcQuerySuite extends QueryTest with BeforeAndAfterAll with OrcTest {
     }
   }
 
+  test("read from multiple input paths") {
+    val path1 = Utils.createTempDir()
+    val path2 = Utils.createTempDir()
+    makeOrcFile((1 to 10).map(Tuple1.apply), path1)
+    makeOrcFile((1 to 10).map(Tuple1.apply), path2)
+    assertResult(20)(read.orc(path1.getCanonicalPath, path2.getCanonicalPath).count())
```

Comment:
`withOrcFile` will get cleaned up faster (e.g. as soon as the test ends rather than program exit).